platform-sdk/consensus-otter-tests/src/testFixtures/java/org/hiero/otter/fixtures/turtle/TurtleNetwork.java

```diff
@@ -235,7 +237,10 @@ public void destroy() throws InterruptedException {
         for (final TurtleNode node : nodes) {
             node.destroy();
         }
+        nodes.clear();
```

Comment:
Why is this needed?

Reply:
Removed. It was a leftover from the previously suggested approach of restarting the turtle network.